sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeContentOptions.java

```diff
@@ -60,4 +61,32 @@ public RecognizeContentOptions setPollInterval(final Duration pollInterval) {
         this.pollInterval = pollInterval == null ? DEFAULT_POLL_INTERVAL : pollInterval;
         return this;
     }
+
+    /**
+     * Get the BCP-47 language code of the text in the document.
+     * See supported language codes here:
+     * <a>
+     * https://docs.microsoft.com/azure/cognitive-services/form-recognizer/language-support.
+     * </a>
+     *
+     * @return the language code for the text in the document.
+     */
+    public String getLanguage() {
+        return language;
+    }
+
+    /**
+     * Set the BCP-47 language code of the text in the document.
+     * See supported language codes here:
+     * <a>
+     * https://docs.microsoft.com/azure/cognitive-services/form-recognizer/language-support.
+     * </a>
+     *
+     * @param language the language code value to set.
```

Comment:
I think it's important to include that auto detect is default and that this forces the document to be analyzed in the given language: https://github.com/Azure/azure-sdk-for-python/blob/feature/formrecognizer2.1/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py#L450

Reply:
Added [here](https://github.com/Azure/azure-sdk-for-java/pull/17387/files#diff-c6dda2f3c94b29f3a3264af6dae5b3d4cb30ac8bf75081dfb9c2dfa211b902ebR243).